data/presets/seamark/offshore_platform.json

```diff
@@ -0,0 +1,40 @@
+{
+    "icon": "temaki-oil_well",
```

Comment:
I am not an expert on this topic - but this icon looks very land-specific

I think that even generic icon nonspecific to anything would be better

Reply:
@k-yle Can you fix the icon? Or at least replace it with some generic one?